gce/Makefile

```diff
@@ -142,6 +145,32 @@ build/calico-cni-url: build/calico-cni-packed-url Makefile local-settings.mk
 	cp build/calico-cni-packed-url build/calico-cni-url
 endif
 
+run-local-prom:
+	# Prometheus needs a config file, but a blank one will do
+	echo "" > $(CURRENT_DIR)/timings/prometheus.yml
+	# Extract the prometheus data saved off from the cloud
+	-tar --overwrite -xvzf $(CURRENT_DIR)/timings/prom_data.tgz -C $(CURRENT_DIR)/timings/
+	# Run prometheus, using the extracted database.
+	docker run --rm --name prometheus -p 9090:9090 -v $(CURRENT_DIR)/timings/prometheus.yml:/etc/prometheus.yml -v $(CURRENT_DIR)/timings/var/log/prom_data:/prometheus prom/prometheus -config.file=/etc/prometheus.yml -storage.local.path=/prometheus &
+
+run-local-grafana:
+	mkdir -p $(CURRENT_DIR)/timings/etc/grafana/dashboards
+	cp local_grafana.ini $(CURRENT_DIR)/timings/etc/grafana/grafana.ini
+	wget -N -P $(CURRENT_DIR)/timings/etc/grafana/ $(GRAFANA_DB_URL)
+	cp templates/grafana.json $(CURRENT_DIR)/timings/etc/grafana/dashboards/grafana-dash.json
+	docker run --rm --name grafana --net=host -v $(CURRENT_DIR)/timings/etc/grafana/:/etc/grafana/ grafana/grafana &
+
+view-diags:
+	make --no-print-directory run-local-prom
+	make --no-print-directory run-local-grafana
+	sleep 2
+	@echo "- Grafana dashboard at http://localhost:3000/dashboard/file/grafana-dash.json"
+	@echo "- Prometheus at http://localhost:9090/"
+
+view-diags-stop:
+	docker stop $(shell docker ps -a -q)
```

Comment:
Won't this stop all docker containers?  I think that would upset people who are using docker for something else!